package.json

```diff
@@ -6,6 +6,9 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "big-js": "^3.1.3",
```

Comment:
- [ ] Hint: I'm commenting here cause the readme file is not included with changes 

- [ ] it would be better if you make the `MIT` link in your readme file  working as expected and it should navigate you to MIT license in order to make your project pro